index.html

```diff
@@ -20,29 +20,30 @@ <h1>Moin, I'm Nils</h1>
       <section>
         <figure>
           <img
+            class="pflanze"
             src="https://user-images.githubusercontent.com/125801955/236850362-d1336789-fe62-4eea-8abc-2af267625983.png"
             alt="Pflanze"
           />
           <figcaption>Aspiring Web Dev at neueFische<a href="https://www.neuefische.de" 
-            target="_blank"><img src="https://images.ctfassets.net/m8n66tuamygx/4hT1EuV1z7nnYGOBXOEWPz/006bf4419464bb53cffcaacb85f84199/metaimage.png"></a></figcaption>
+            target="_blank"><img src="https://images.ctfassets.net/m8n66tuamygx/4hT1EuV1z7nnYGOBXOEWPz/006bf4419464bb53cffcaacb85f84199/metaimage.png" class="nflogo"></a></figcaption>
         </figure>
       </section>
-        <section id="bio">
+        <section class="bio">
```

Comment:
um in deiner navigation hierhin zu verlinken musst du der section trotzdem die ID geben, sonst funktioniert `href="#bio` nicht 
Vorschlag:
` <section class="bio" id="bio">`
benutz die id einfach nur um darauf zu verlinken und als "anker" auf der seite, aber nutzen die Klasse zum CSS-Stylen

Reply:
Als ich die ID's mit den Class's ersetzt habe, habe ich vergessen das anzupassen. Ich habe die #bio, ... jetzt mit .bio, ... ersetzt. Danke!